<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ControlsDockWidget</class>
 <widget class="QDockWidget" name="ControlsDockWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>258</width>
    <height>736</height>
   </rect>
  </property>
  <property name="features">
   <set>QDockWidget::DockWidgetFloatable|QDockWidget::DockWidgetMovable</set>
  </property>
  <property name="windowTitle">
   <string>NetRoute Controls</string>
  </property>
  <widget class="QWidget" name="dockWidgetContents">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QGroupBox" name="displayGroup">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="title">
       <string>Display</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="0" colspan="2">
        <widget class="QCheckBox" name="highlightStartCheck">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Highlight start node</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="1" column="0" colspan="2">
        <widget class="QCheckBox" name="highlightEndCheck">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Highlight end node</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="2" column="0" colspan="2">
        <widget class="QCheckBox" name="highlightPathCheck">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Highlight path</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="layoutGroup">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="title">
       <string>Layout</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <layout class="QFormLayout" name="formLayout">
         <item row="0" column="0">
          <widget class="QLabel" name="label">
           <property name="text">
            <string>Max. Iterations:</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QSpinBox" name="maxIterationsSpinner">
           <property name="prefix">
            <string/>
           </property>
           <property name="maximum">
            <number>5000</number>
           </property>
           <property name="value">
            <number>100</number>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QDoubleSpinBox" name="layoutToleranceSpinner">
           <property name="decimals">
            <number>4</number>
           </property>
           <property name="maximum">
            <double>1.000000000000000</double>
           </property>
           <property name="singleStep">
            <double>0.000100000000000</double>
           </property>
           <property name="value">
            <double>0.001000000000000</double>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>Tolerance</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QPushButton" name="randomLayoutButton">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>Random Layout</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="springLayoutButton">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>Spring Layout</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="groupBox_2">
      <property name="title">
       <string>Network Specification</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="0" column="0" colspan="2">
        <widget class="QPlainTextEdit" name="problemSpecEdit">
         <property name="lineWrapMode">
          <enum>QPlainTextEdit::NoWrap</enum>
         </property>
        </widget>
       </item>
       <item row="1" column="0" colspan="2">
        <widget class="QPushButton" name="pasteAndRouteButton">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Paste and Route</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QPushButton" name="routeButton">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Route</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QPushButton" name="clearSpecButton">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Clear</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="groupBox_3">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
      <property name="title">
       <string>Actions</string>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="2" column="0">
        <widget class="QPushButton" name="exitButton">
         <property name="text">
          <string>Exit</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QPushButton" name="clearNetworkButton">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Clear Network</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QPushButton" name="generateReportButton">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Generate Text Report</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <tabstops>
  <tabstop>highlightStartCheck</tabstop>
  <tabstop>highlightEndCheck</tabstop>
  <tabstop>highlightPathCheck</tabstop>
  <tabstop>maxIterationsSpinner</tabstop>
  <tabstop>layoutToleranceSpinner</tabstop>
  <tabstop>randomLayoutButton</tabstop>
  <tabstop>springLayoutButton</tabstop>
  <tabstop>problemSpecEdit</tabstop>
  <tabstop>pasteAndRouteButton</tabstop>
  <tabstop>routeButton</tabstop>
  <tabstop>clearSpecButton</tabstop>
  <tabstop>clearNetworkButton</tabstop>
  <tabstop>generateReportButton</tabstop>
  <tabstop>exitButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
